raco/myrial/scanner.py

```diff
@@ -17,7 +17,7 @@
 
 builtins = ['EMPTY', 'WORKER_ID', 'SCAN', 'COUNTALL', 'COUNT', 'STORE',
             'DIFF', 'CROSS', 'JOIN', 'UNIONALL', 'INTERSECT', 'DISTINCT',
-            'LIMIT', 'SINK']
+            'LIMIT', 'SINK', 'SAMPLEWRSCAN', 'SAMPLEWORSCAN']
```

Comment:
These need less opaque names. Why not make just SAMPLESCAN with an extra argument?